prereise/gather/demanddata/transportation_electrification/tests/test_immediate_charging_integration.py

```diff
@@ -14,8 +15,118 @@
 )
 
 
+def test_immediate_charging_ldv_2days():
```

Comment:
Just curious, why we do 2-day checking for `ldv` here, whereas `mdv` and `hdv` 1-day?

Reply:
I see. That makes sense.